examples/helloworld/BUILD

```diff
@@ -111,6 +131,7 @@ sh_test(
     ],
     data = [
         ":canary.show",
+        ":gitops_custom_path",
```

Comment:
Please, remove this line. You are not assertign anything from the `gitops_custom_path` output in `k8s_deploy_test.sh`.

Reply:
cool, done, wasn't sure what this was for